packages/@glimmer/test-helpers/lib/suites/initial-render.ts

```diff
@@ -256,6 +256,50 @@ export class InitialRenderSuite extends RenderTest {
     this.assertStableRerender();
   }
 
+  @test
```

Comment:
These tests are similar (copy&paste) to [Unquoted attribute null values do not disable](https://github.com/glimmerjs/glimmer-vm/pull/841/files#diff-c826bfbc3bab3c7ef42bc6f7884b1ebbR238), but updated for other `falsy` values.